### DISCUSSION TITLE
Inverter polling frequency

### DISCUSSION THREAD

**Discussion by AnKloss** [wlcrs/huawei_solar#305] [General]
Hello,
this code from your Document has to be put into with file in Home Assistant?

mfg
Andreas

- alias: "Huawei Solar inverter data polling"
  trigger:
    - platform: time_pattern
      hours: "*"
      minutes: "*"
      seconds: "/10"
  action:
    - service: homeassistant.update_entity
      target:
        entity_id: sensor.daily_yield

**Comment by jires** [comment]
You need to put it into automations.yaml
